manifests/base/deployment.yaml

```diff
@@ -97,11 +97,11 @@ spec:
           failureThreshold: 3
         resources:
           limits:
-            cpu: 100m
-            memory: 256Mi
+            cpu: 500m
+            memory: 1Gi
           requests:
-            cpu: 100m
-            memory: 256Mi
+            cpu: 1000m
```

Comment:
You're requesting 1000 but limits are 500? I think you got these backwards @VaishnaviHire

Reply:
Yes I did. Thanks!! Updated now